actions/poll_status.yaml

```diff
@@ -3,7 +3,7 @@ name: poll_status
 runner_type: run-python
 description: Polls an URL until the web service says that the long process has finished
 enabled: true
-entry_point: lib/poll_status.py
+entry_point: poll_status.py
```

Comment:
Is there any specific reasons for why you moved this away from the `lib` subfolder btw?